src/NeuralNetDiffEq.jl

```diff
@@ -56,13 +56,35 @@ function Base.show(io::IO, A::KolmogorovPDEProblem)
   println(io,"Sigma")
   show(io , A.sigma)
 end
+struct OptimalStoppingProblem{ Mu, Sigma, G , U0 , T ,P} <: DiffEqBase.DEProblem
```

Comment:
More generally, this is just an SDE problem with a value function `g`, so I would just make this `SDEProblem` and `g`.